### PR TITLE
Feat: now have 103 and 426 http-codes

### PR DESCRIPTION
**Pull request by angellee177**
fix issue for this ticket [https://github.com/prettymuchbryce/http-status-codes/issues/20](url)

### CODE REVIEW

**Comment by angellee177** [comment]
@prettymuchbryce please kindly review this PR. :)

**Comment by angellee177** [comment]
@prettymuchbryce is there anything i need to changes or fix?

**Comment by angellee177** [comment]
@JameelKhan9 @sebdeckers @narqo @tlhunter is there anyone can review this PR?

**Comment by pacifiquem** [comment]
@angellee177 map this issue (  #20 ) to this pull request so that it'll be closed as soon as this pull request is merged.
also everything looks good @prettymuchbryce it can be merged.

**Review by pacifiquem — APPROVED** [review]
@angellee177 map this issue ( https://github.com/prettymuchbryce/http-status-codes/issues/20 ) to this pull request so that it'll be closed as soon as this pull request is merged.
also everything looks good @prettymuchbryce it can be merged.

**Comment by angelsupp** [comment]
thank you for reviewing my PR @pacifiquem :) can you help me to merge the PR?

**Comment by pacifiquem** [comment]
No @angelsupp we've to wait for @prettymuchbryce or any other maintainer to merge it. Since everything looks good let's hope they'll merge it soon :).

**Comment by angelsupp** [comment]
thanks a lot @pacifiquem 😆

**Comment by prettymuchbryce** [comment]
Thank you and apologies for my delayed response. It looks like HTTP 521 is not an official status code, but instead is specific to CloudFlare. The documentation linked in this PR
[here](https://datatracker.ietf.org/doc/html/rfc7504#section-3) is for an unrelated SMTP (not HTTP) error.

I would prefer not to include codes without an official RFC. If we can remove HTTP 521 from this PR then I will happily merge the PR 103 and 426 look good to me!

File: codes.json
Comment (pacifiquem) [inline codes.json:528]: It looks like HTTP 521 is not an official status code, but instead is specific to CloudFlare. The documentation linked in this PR [here](https://datatracker.ietf.org/doc/html/rfc7504#section-3) is for an unrelated SMTP (not HTTP) error.

@angellee177 or @angelsupp remove this status code ( 521 ) and this PR will be merged as suggested by @prettymuchbryce

**Review by pacifiquem — CHANGES_REQUESTED** [review]
@angellee177 @angelsupp  remove 521 status code as suggested by @prettymuchbryce and this PR will be merged. Also remember to map issue #20 so that it will be closed immediately as 103 status code will be added.

**Comment by pacifiquem** [comment]
Closes #20

**Comment by angelsupp** [comment]
let me remove the 521 status code @pacifiquem @prettymuchbryce

**Comment by angelsupp** [comment]
done @prettymuchbryce @pacifiquem 😆
[here](https://github.com/prettymuchbryce/http-status-codes/pull/89/commits/7b84773c9da8e9f92fe364d8991362490a2e68c2) is the commit.

**Review by pacifiquem — APPROVED** [review]
LGTM :+1:

**Comment by pacifiquem** [comment]
@prettymuchbryce let's merge it.

**Comment by pacifiquem** [comment]
Thank you @angelsupp for your contribution.

**Pull request merged**
